src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -167,15 +167,15 @@ private static String generateDescribableHelp(Descriptor d) {
         } else {
             Set<String> symbols = SymbolLookup.getSymbolValue(d);
             if (!symbols.isEmpty()) {
-                StringBuilder mkDesc = new StringBuilder(header(3)).append(" +").append(symbols.iterator().next()).append("+: ").append(d.getDisplayName()).append("\n");
+                StringBuilder mkDesc = new StringBuilder(header(3)).append(" `").append(symbols.iterator().next()).append("`: ").append(d.getDisplayName()).append("\n");
                 try {
                     mkDesc.append(generateHelp(new DescribableModel<>(d.clazz), true)).append("\n\n");
                 } catch (Exception ex) {
                     ex.printStackTrace();
-                    mkDesc.append("+").append(ex).append("+\n\n");
+                    mkDesc.append("`+").append(ex).append("+`\n\n");
```

Comment:
Should that plus be removed as well?

Reply:
@kwhetstone The backtick just displays as monospace, backtick-plus shows monospace and passes the literal text through.  I'm not sure what text might be passed in the exception's toString, but this will make it less likely to cause generation problems. 

For example, (backtick)'Exception: Failed to find sting "{help-me}" in output.'(backtick) will be interpreted on the asciidoc side as containing an attribute `help-me` (which of course doesn't exist). Backtick-plus prevents that. 

I felt it wasn't needed in the title because the string pretty well understood.

Reply:
Cool, thanks for the explanation.